lib/ansible/modules/cloud/azure/azure_rm_manageddisk_facts.py

```diff
@@ -112,8 +116,7 @@
         tags:
             description:
                 - Tags to assign to the managed disk.
-            type: dict
-            sample: { "tag": "value" }
+                - Format tags as 'key' or 'key:value'.
```

Comment:
I'd suggest keeping the old lines 115-116. This section describes the return data, so knowing that it's a dict and having the sample would be useful. The user cannot format the tags in the return data.